yarn.lock

```diff
@@ -404,7 +404,7 @@ bluebird@3.5.1:
   resolved "https://registry.yarnpkg.com/bluebird/-/bluebird-3.5.1.tgz#d9551f9de98f1fcda1e683d17ee91a0602ee2eb9"
   integrity sha512-MKiLiV+I1AA596t9w1sQJ8jkiSr5+ZKi0WKrYGUn6d1Fx+Ij4tIj+m2WMQSGczs5jZVxV339chE8iwk6F64wjA==
 
-body-parser@1.19.0:
+body-parser@1.19.0, body-parser@^1.19.0:
```

Comment:
Please remove the yarn.lock on your local env and run `yarn` again.

Reply:
Should I include the yarn lock in the git ignore?

Reply:
No. The yarn.lock is required to make sure all dependencies are installed with their correct version.